### ISSUE TITLE
About Frame mAP

### ISSUE THREAD

**Issue by zyx-cv** [MCG-NJU/MultiSports#25]
Hello， I would like to confirm if it is necessary to inference each frame in the validation set video and get the result when calculating the frame map. If only the frames containing ground truth frames are inferred, the final precision would be falsely high, right?

**Comment by yixuanli98** [comment]
Yes, you need to inference each frame in the validation set. The model need to identify if there exists actions in current frame.
